views/jobs.ejs

```diff
@@ -0,0 +1,37 @@
+<h2>Jobs List</h2>
+<a href="/jobs/new">
+  <button type="button">Add New Job</button>
+</a>
+<table id="jobs-table">
+  <tr id="jobs-table-header">
+    <th>Company</th>
+    <th>Position</th>
+    <th>Status</th>
+    <th colspan="2">Actions</th>
+  </tr>
+  <% if (jobs && jobs.length) { %>
+    <% jobs.forEach((job) => { %>
+      <tr>
+        <td><%= job.company %></td>
+        <td><%= job.position %></td>
+        <td><%= job.status %></td>
+        <td>
+            <a href="/jobs/edit/<%= job.id %>">
+              <button type="button">edit</button>
+            </a>
+        </td>
+        <td>
+          <form action="/jobs/delete/<%= job.id %>" method="POST" style="display:inline;">
+            <input type="hidden" name="_method" value="DELETE">
+            <button type="submit">Delete</button>
+            <input type="hidden" name="_csrf" value="<%= _csrf %>" />
```

Comment:
Requires change:
Please use consistent naming for your hidden csrf input's value. In this case, it should be `value="<%= csrfToken %>"` as mentioned in a previous review comment!